firefox-ios/Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -31,6 +31,7 @@ enum NimbusFeatureFlagID: String, CaseIterable {
     case nightMode
     case preferSwitchToOpenTabOverDuplicate
     case reduxSearchSettings
+    case remoteTabManagement
```

Comment:
```suggestion
    case closeRemoteTabs
```

Nitpick: How would you feel about changing this flag's name to `closeRemoteTabs`? `remote-tab-management` is the name of the Nimbus feature; `close-tabs` is the actual variable.

If we introduce other variables in the `remote-tab-management` feature (`close-all-tabs`, maybe? `move-tab`? 😅), it might be a little confusing to have the `remoteTabManagement` flag actually just mean "close tabs".

Reply:
That definitely makes more sense! Updated